GiEnJul/ClientApp/src/components/InputFields/Validators/InputValidator.tsx

```diff
@@ -85,7 +86,8 @@ const InputValidator: FC<InputValidatorProps> = (
     }
 
     const validate = () => {
-        let errorMsg = "";
+        let errorMsg: string | undefined = undefined;
+        let isAnyError = false;
         for (let i = 0; i < validators.length; i++) {
```

Comment:
if validators is an array of values its better to use 
validators.map(validator => { // do something })

for better readability